app/models/generic_event/per_handover.rb

```diff
@@ -0,0 +1,5 @@
+class GenericEvent
+  class PerHandover < GenericEvent
+    include PersonEscortRecordEventValidations
```

Comment:
do the details attributes need to be specified too? - i.e.. 
```
{
          'status': status,
          'handover_details': handover_details,
          'handover_occurred_at': occurred_timestamp.iso8601,
}
```

Reply:
No, not in this case. We decided to keep the choice of `handover_details` in the front end only to allow scope for these to vary over time. So the `details` in the event only contains the `handover_details` from the PER. We don't need a status attribute in there and the `handover_occurred_at` timestamp is stored in the event `occurred_at` attribute. As we're not enforcing strong typing in the API and support any JSON structure for `handover_details` there's no need to explicitly list the details here.

Reply:
👍  jolly good